antivir/models/mail_message.py

```diff
@@ -20,21 +20,21 @@
 #
 ##############################################################################
 
-from openerp.osv import osv, fields
-from openerp.tools.translate import _
+from odoo import models, fields, api, _
 from .exceptions import VirusFound
 
-class mail_message(osv.Model):
+
+class MailMessage(models.Model):
     _inherit = 'mail.message'
 
-    def create(self, cr, uid, values, context=None):
+    @api.model
+    def create(self, vals):
         try:
-            result = super(mail_message, self).create(cr, uid, values, context=context)
+            result = super(MailMessage, self).create(vals)
```

Comment:
return super(MailMessage, self).create(vals)

Reply:
I agree with you.